src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ quarkus.log.console.format=%d{HH:mm:ss} %-5p [%c{2.}] (%t) %s%e%n
 quarkus.log.console.level=DEBUG
 quarkus.log.console.color=false
 quarkus.log.category."io.quarkus".level=INFO
-quarkus.log.category."com.redhat.labs".level=${OMP_LOGGING:DEBUG}
+quarkus.log.category."com.redhat.labs".level=${OMP_LOGGING:INFO}
```

Comment:
Are we ready for INFO level default? I think in all envs the default is used so any debug logging will not be available without confi in test / stage / prod

Reply:
Returned the value to DEBUG.  Changed it while testing the performance parts, but with the logging level at INFO vs DEBUG it was negligible.